MAUI/Cartesian-Charts/Axis/Axislabels.md

```diff
@@ -11,6 +11,42 @@ documentation: ug
 
 Axis labels are used to show the units or measures or category value of axis to visualize the data user friendly. It will be generated based on the range and the values binded to [XBindingPath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSeries.html#Syncfusion_Maui_Charts_ChartSeries_XBindingPath) or [YBindingPath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.XYDataSeries.html#Syncfusion_Maui_Charts_XYDataSeries_YBindingPath) properties of series.
 
+## Positioning the labels
+
+The [LabelsPosition]() property is used to position the axis labels inside or outside the chart area. By default, [LabelsPosition]() is positioned `AxisElementPosition.Outside` the chart area.
```

Comment:
Improve the default value like follows -https://help.syncfusion.com/maui/cartesian-charts/axis/axislabels#edge-labels-drawing-mode

Reply:
Updated the content as present in the link